lib/screens/landing_page/landing_page.dart

```diff
@@ -0,0 +1,269 @@
+                                toptext2: 30,
+                                bottomtext3: 0,
+                                righttext3: 0,
+                                color: Colors.pink,
+                                textColor: light,
+                                text2: 'BUY & SELL',
+                                text3:
+                                    'Buy and sell your daily household and agri needs!',
+                              ),
+                            ],
+                          )
+                        ],
+                      ),
+                      SizedBox(height: 5),
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Column(
+                            children: [
+                              CustomCard(
+                                text: 'E-Learning',
+                                image: 'assets/images/online-learning.png',
+                                onClick: () {
+                                  print('E-Learning');
+                                },
+                                top: 0,
+                                right: 0,
+                                bottomtext: 0,
+                                bottomtext2: 2,
+                                toptext3: 0,
+                                lefttext3: 0,
+                                color: greenCustom,
+                                textColor: light,
+                                text2: 'ENROLL & ENJOY',
+                                text3: 'Learn from the comfort of your home!',
+                              ),
+                            ],
+                          ),
+                          // SizedBox(width: 20),
+                          Column(
+                            children: [
+                              CustomCard(
+                                text: 'AGRI-INSURANCE',
+                                image: 'assets/images/chat.png',
+                                onClick: () {
+                                  print('Support');
+                                },
+                                top: 0,
+                                left: 0,
+                                bottomtext: 0,
+                                bottomtext2: 5,
+                                toptext3: 0,
+                                righttext3: 0,
+                                color: Colors.purple,
+                                textColor: light,
+                                text2: 'APPLY AND UPDATE',
+                                text3:
+                                    'Worry no more about your application and claims!',
+                              ),
+                            ],
+                          )
+                        ],
+                      ),
+                      SizedBox(height: 10),
```

Comment:
Kindly remove this button, it is at this moment not, necessary.

Reply:
alright